ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -369,6 +369,17 @@ objects:
           setup-google-cloud-sdk
         fi
 
+        if [[ "${ENABLE_FIPS}" == true ]]; then
+          oc --insecure-skip-tls-verify --request-timeout=5s get nodes -o jsonpath --template '{range .items[*]}{.metadata.name}{"\n"}{end}' > /tmp/nodes
+          while read p; do
+            oc debug node/"$p" -- cat /proc/sys/crypto/fips_enabled > /tmp/enabled
+            if [[ $(< /tmp/enabled) != "1" ]]; then
+              echo fips not enabled in node "$p", exiting
+              exit 1
+            fi
+          done </tmp/nodes
+        fi
+
```

Comment:
@mrunalp this is what i came up with quickly for a fips check, works locally let's see if it runs : )
can you point me to any other way to verify fips is enabled? thanks

Reply:
This looks good. @yuqi-zhang do we also want to add a check to look at machine config pools?

Reply:
The MCP doesn't have anything explicit that tells you whether FIPS is enabled or not. For now I guess the existence of `99-master-fips` and `99-worker-fips` determines whether a FIPS MC exists. You would have to take the rendered configs (e.g. `rendered-master-fc7c41b655be129491eadeae4f1bd2d9`) and view whether the config has `FIPS=true`. For now I think this check is enough. The MCO has built in checks to fail if the system FIPS is not what we set.

Reply:
ok, for now I've added the check for `cat /proc/sys/crypto/fips_enabled` can add other in followup